other/install_2-1_mysql.sql

```diff
@@ -1179,9 +1179,9 @@ CREATE TABLE {$db_prefix}user_likes (
 CREATE TABLE {$db_prefix}mentions (
   content_id INT DEFAULT '0',
   content_type VARCHAR(10) DEFAULT '',
-  id_mentioned INT DEFAULT 0,
-  id_member INT(10) UNSIGNED NOT NULL DEFAULT 0,
-  `time` INT NOT NULL DEFAULT 0,
+  id_mentioned INT DEFAULT '0',
```

Comment:
MySQL does not accept default 0.  It requires default '0'.   

MySQL does a lot of things that are not ansi-standard.  That's one of the reasons folks like pg.  

I think pg just tolerates it...